ui/apps/platform/src/Containers/Vulnerabilities/types.ts

```diff
@@ -66,7 +66,15 @@ export function isValidWorkloadEntityTab(value: unknown): value is WorkloadEntit
     return workloadEntityTabValues.some((tab) => tab === value);
 }
 
-export type EntityTab = WorkloadEntityTab;
+export const nodeEntityTabValues = ['CVE', 'Node'] as const;
+
+export type NodeEntityTab = (typeof nodeEntityTabValues)[number];
+
+export function isValidNodeEntityTab(value: unknown): value is NodeEntityTab {
```

Comment:
I noticed this function and `isValidWorkloadEntityTab` aren't being used. Will they be used later or did it turn out they weren't needed?

Reply:
Hmm I guess they aren't needed. I think `isValidWorkloadEntityTab` may have been used at some point, but no longer is.

I can clean these out in a follow up.